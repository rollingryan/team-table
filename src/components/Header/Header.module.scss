.header {
  margin: 2rem 0;

  &__content {
    display: flex;
    flex-flow: row nowrap;
    justify-content: space-between;
    align-items: flex-end;
  }

  &__title,
  &__subtitle {
    width: 50%;
    margin: 0;
  }

  &__title {
    display: flex;
    flex-flow: column nowrap;
    align-items: flex-end;
    padding-right: 0.5rem;
    text-transform: uppercase;
    line-height: 1;
  }

  &__subtitle {
    text-align: left;
    padding-left: 0.5rem;
    text-transform: lowercase;
  }
}
