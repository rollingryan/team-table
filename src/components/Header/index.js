import React from 'react'
import { Row, Col } from 'react-bootstrap'
import styles from './styles.module.scss'

const Header = () => (
  <Row className={styles.header}>
    <Col className={styles.header__content}>
      <h1 className={styles.header__title}>
        <span>Team</span>
        <span>Scans</span>
      </h1>
      <h4 className={styles.header__subtitle}>By month or by week</h4>
    </Col>
  </Row>
)

export default Header
