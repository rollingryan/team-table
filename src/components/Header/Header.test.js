import React from 'react'
import renderer from 'react-test-renderer'
import Header from '.'

test('Header', () => {
  const component = renderer.create(<Header />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
