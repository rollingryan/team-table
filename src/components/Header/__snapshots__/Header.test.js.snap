// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<div
  className="header row"
>
  <div
    className="header__content col"
  >
    <h1
      className="header__title"
    >
      <span>
        Team
      </span>
      <span>
        Scans
      </span>
    </h1>
    <h4
      className="header__subtitle"
    >
      By month or by week
    </h4>
  </div>
</div>
`;
